### ISSUE TITLE
resize the window,proc is crashed on windows

### ISSUE THREAD

**Issue by lld1995** [gui-cs/Terminal.Gui#2730]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://github.com/gui-cs/Terminal.Gui/assets/17978390/7dc46a2d-0e34-45a2-a31d-879882ed4a26)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
1.set Application.UseSystemConsole = true;
2.Application.Run<MainWindow>();
3.continuous use mouse pointer to resize the mainwindow
4.it crashed!

**Comment by BDisp** [comment]
I have `Windows 11` and it don't crash with `Windows Terminal` and `Windows Host Console`. But you are getting the crash in the `ConsolePal.Windows.cs` which belong to the `System.Console.dll` and it doesn't be handled on the `Terminal.Gui`. Maybe is better you open an issue on the `Microsoft.dotnet` repo.

**Comment by tznind** [comment]
Could be the same issue as https://github.com/gui-cs/Terminal.Gui/issues/2714 but manifesting differently?

**Comment by BDisp** [comment]
> Could be the same issue as #2714 but manifesting differently?

I don't think so because the #2714 happens with the `WindowsDriver` and this one happens with the `NetDriver`. But I could consider the same issue if the `System.Console.dll` is based on the "Win32 API".

**Comment by lld1995** [comment]
> > Could be the same issue as #2714 but manifesting differently?
> 
> I don't think so because the #2714 happens with the `WindowsDriver` and this one happens with the `NetDriver`. But I could consider the same issue if the `System.Console.dll` is based on the `Win32 API`, because both issues happens on `Windows 10`.

i also don't think so,because if i remove Application.UseSystemConsole = true; ,it not crashed,i think i should also mention that one things
i use scrollview,and set this.scrollView.ContentSize when main window resized

**Comment by BDisp** [comment]
> i also don't think so,because if i remove Application.UseSystemConsole = true; ,it not crashed,i think i should also mention that one things
> i use scrollview,and set this.scrollView.ContentSize when main window resized

Did you follow all the steps indicated on the #2714?

**Comment by lld1995** [comment]
> > i also don't think so,because if i remove Application.UseSystemConsole = true; ,it not crashed,i think i should also mention that one things
> > i use scrollview,and set this.scrollView.ContentSize when main window resized
> 
> Did you follow all the steps indicated on the #2714?

not all,but sometimes i have the same pr

Sometimes method below throws same exception **No process is on the other end of the pipe**. while playing with corners really fast.

**Comment by BDisp** [comment]
> not all,but sometimes i have the same pr
> 
> Sometimes method below throws same exception **No process is on the other end of the pipe**. while playing with corners really fast.

Thanks. For sure it's related from the same issue on the `Windows 10` with the `Win32 API`. Only the `dotnet` team can fix this. I recommend open an issue there referencing this both issues.

**Comment by lld1995** [comment]
> > not all,but sometimes i have the same pr
> > Sometimes method below throws same exception **No process is on the other end of the pipe**. while playing with corners really fast.
> 
> Thanks. For sure it's related from the same issue on the `Windows 10` with the `Win32 API`. Only the `dotnet` team can fix this. I recommend open an issue there referencing this both issues.

ok,thanks.can you paste the dotnet issue link there?

**Comment by tznind** [comment]
> No process is on the other end of the pipe.  [...] Only the dotnet team can fix this. 

This might be true but I am hopeful we can still find a workaround or a way to reduce the frequency of this crash.  Since it seems to be related to have fast you resize the window we might be able to throttle the resize events on Windows or somehow catch this specific exception?

**Comment by BDisp** [comment]
> This might be true but I am hopeful we can still find a workaround or a way to reduce the frequency of this crash. Since it seems to be related to have fast you resize the window we might be able to throttle the resize events on Windows or somehow catch this specific exception?

In the case of `NetDriver` it can't be fixed because I experienced this issue during long time, even on `Windows 11` when the height is less than 1,  although I tried to avoid it in code, but without success. I think there was a fix in the library for the `Windows 11` and not for `Windows 10`.
@tznind can you open an issue there please?

**Comment by BDisp** [comment]
@lld1995 can you confirm if you are using the `Windows 10 Version 22H2`?

**Comment by Decimation** [comment]
I have the same problem here.

**Comment by lld1995** [comment]
> 

win10 22H2 19045.3086
